src/EventStore.Core/Services/ElectionsService.cs

```diff
@@ -100,6 +100,10 @@ public class ElectionsService : IHandle<SystemMessage.BecomeShuttingDown>,
 			if (memberInfo.IsReadOnlyReplica) {
 				throw new ArgumentException("Read-only replicas are not allowed to run the Elections service.");
 			}
+			if (leaderElectionTimeout.TotalSeconds < 1) {
+				throw new ArgumentOutOfRangeException(nameof(leaderElectionTimeout),
+					$"{nameof(leaderElectionTimeout)} should be greater than 1 second.");
```

Comment:
```suggestion
					$"{nameof(leaderElectionTimeout)} should not be less than 1 second.");
```
Just a small precision to the error message